machine-learning/src/main.py

```diff
@@ -0,0 +1,61 @@
+import os
+from flask import Flask, request
+from transformers import pipeline
+
+
+server = Flask(__name__)
+
+
+classifier = pipeline(
+    task="image-classification",
+    model="microsoft/resnet-50"
+)
+
+detector = pipeline(
+    task="object-detection",
+    model="hustvl/yolos-tiny"
+)
+
+
+# Environment resolver
+is_dev = os.getenv('NODE_ENV') == 'development'
+server_port = os.getenv('MACHINE_LEARNING_PORT') or 3003
+
+
+@server.route("/ping")
+def ping():
+    return "pong"
+
+
+@server.route("/object-detection/detect-object", methods=['POST'])
+def object_detection():
+    assetPath = request.json['thumbnailPath']
+    return run_engine(detector, assetPath), 201
+
+
+@server.route("/image-classifier/tag-image", methods=['POST'])
+def image_classification():
+    assetPath = request.json['thumbnailPath']
+    return run_engine(classifier, assetPath), 201
+
+
+def run_engine(engine, path):
```

Comment:
Fine for now, but when we go to add CLIP etc it's probably worth extracting the actual ML logic to a separate service layer.